bin/dipy_median_ostu

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
```

Comment:
The name is dipy_median_otsu
Also it is not executable. You need to change the permissions of the file. Or we have to make it 
executable when we install it. Not sure.